src/Scene/NodeProcess.js

```diff
@@ -415,24 +402,19 @@ NodeProcess.prototype.processNode = function(node, camera, params) {
         if (sse && params.tree.canSubdivideNode(node)) {
             // big screen space error: subdivide node, display children if possible
             this.subdivideNode(node, camera, params);
-        }
-
-        if (!hidden)  {
+        } else if (!hidden && node.downScaledLayer(1)) {
```

Comment:
Don't use `1` magic value.
Also, are you not interested in refining elevation layers?